packages/block-editor/src/components/link-control/style.scss

```diff
@@ -55,6 +55,7 @@ $preview-image-height: 140px;
 	.components-base-control__label {
 		margin-right: $grid-unit-20;
 		margin-bottom: 0;
+		width: 29px; // align with search results.
```

Comment:
should this be `min-width`? I'm thinking of other languages that will have longer labels

Reply:
✅